pkg/controller/backupinfrastructure/backup_infrastructure_control.go

```diff
@@ -212,6 +220,12 @@ func (c *defaultControl) ReconcileBackupInfrastructure(obj *gardenv1beta1.Backup
 		return updateErr
 	}
 
+	kutil.TryBackupInfrastructureAnnotations(op.K8sGardenClient.GardenClientset(), retry.DefaultRetry, obj.ObjectMeta,
```

Comment:
Error checking is missing here.

Reply:
Btw shouldn't this be `TryUpdateBackupInfrastructureAnnotations`?